web/web.go

```diff
@@ -76,6 +76,12 @@ func IsApiCall(a *app.App, r *http.Request) bool {
 	return strings.Index(r.URL.Path, path.Join(subpath, "api")+"/") == 0
 }
 
+func IsWebhookCall(a *app.App, r *http.Request) bool {
+	subpath, _ := utils.GetSubpathFromConfig(a.Config())
+
+	return strings.Index(r.URL.Path, path.Join(subpath, "hooks")+"/") == 0
```

Comment:
I guess this matches the above code, but wouldn't using `strings.HasPrefix` be more idiomatic here?

Reply:
Yes, you are right, I' going to change it for both functions.

Reply:
Changed! :)